README.md

```diff
@@ -4,10 +4,11 @@ Electron app with React.js
 ![](./docs/screenshot.png)
 # How to develop
 1. `npm i`
-2. `npm start`
-3. open application in build directory
-4. add some react components and styles
-5. reload application (⌘+R)
+2. `gulp atom`
```

Comment:
`gulp` is a only dev-dependency. So user that not installed gulp command system-widely, can't run `gulp atom`.
It should run in npm scripts.